src/main/java/ogc/rs/processes/ProcessesRunnerImpl.java

```diff
@@ -65,10 +65,14 @@ public class ProcessesRunnerImpl implements ProcessesRunnerService {
    * @return a {@code DataFromS3} instance
    */
   private DataFromS3 getS3Object(JsonObject config) {
-    S3Config s3conf = new S3Config.Builder().endpoint(config.getString("awsEndPoint"))
-        .region(config.getString("s3Region")).accessKey(config.getString("awsAccessKey"))
-        .secretKey(config.getString("awsSecretKey")).bucket(config.getString("s3BucketUrl"))
-        .pathBasedAccess(config.getBoolean("s3PathBasedAccess")).build();
+    S3Config s3conf = new S3Config.Builder()
```

Comment:
Does the S3Config config differ in ProcessVerticle and other Verticles? Can't we pass around the S3Config Object between Verticles?

Reply:
You are right. It was done like this for this PR to have minimal changes to make things work. In the multiple buckets PR, the `ProcessVerticle` will take an object that has a list of `S3Config` objects. Each process can work on the bucket of it's choice by choosing the appropriate config.